src/main/scala/is/hail/expr/ir/functions/MathFunctions.scala

```diff
@@ -22,6 +22,20 @@ object MathFunctions extends RegistryFunctions {
 
   def ceil(x: Double): Double = math.ceil(x)
 
+  def mod(x: Float, y: Float): Float = {
+    val t = x % y
```

Comment:
Do we need to check if `y == 0` for divide by zero error?

Reply:
Also, this implementation is different than the one in the FunctionRegistry now. I can't tell if they're equivalent.

Reply:
I copied and pasted from the function registry and then @cseed told me to do this instead. I'll look at it some more